src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -399,11 +410,13 @@ public void run() {
                                 dbq.cbc.error("couldn't delete database");
                             }
                         } catch (Exception e) {
+                            logger.logError("Error deleteDatabaseNow: " + e.getMessage(), "SQLite");
                             Log.e(SQLitePlugin.class.getSimpleName(), "couldn't delete database", e);
                             dbq.cbc.error("couldn't delete database: " + e);
                         }
                     }                    
                 } catch (Exception e) {
+                    logger.logError("Error closeDatabaseNow: " + e.getMessage(), "SQLite");
```

Comment:
This will probably be hit every time there is an upgrade, since we try to close the connection when the web page is unloaded and most of the times it fails because there is a running transaction. Do we want to produce all these logs?

Reply:
"The error appears to happen when the app is updating itself, possibly when opening the app from a push notification." Yes, we want to know if there is some kind of issue related with concurrency or if is just an unexpected error closing the database. We also talk with the client saying that for now this plugin should only be used "in house".